src/test/e2e/00_use_cli_test.go

```diff
@@ -63,6 +63,15 @@ func TestUseCLI(t *testing.T) {
 	assert.NoError(t, err, stdOut, stdErr)
 	assert.Contains(t, stdOut, "nginx:1.16.0", "The chart image should be found by Zarf")
 
+	// Test `zarf prepare find-images` on a chart that has a `kubeVersion` declaration greater than the default (v1.20.0)
+	_, _, err = e2e.ExecZarfCommand("prepare", "find-images", "src/test/test-packages/00-kube-version-override")
+	assert.Error(t, err)
```

Comment:
This is where your tests are failing - the logic here makes sense but because we aren't actually returning a non-zero exit code the command itself doesn't fail.  This could be better written as a require.Contains to look for the warning message in stdErr instead.

Also I didn't realize this file was still using `assert` over `require` - `require` will fail immediately and is generally what we have been switching to because it is easier to debug than `assert` (they share the same interfaces so it should be as easy as a find and replace to fix that).

Reply:
Good point!